src/Network/Network.Test/ScenarioTests/VirtualNetworkTests.cs

```diff
@@ -50,8 +50,8 @@ public void TestVirtualNetworkSubnetCRUD()
             TestRunner.RunTestScript("Test-subnetCRUD");
         }
 
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Fact(Skip = "Authentication failed for auxiliary token: The '1' auxiliary tokens contains duplicates which are from the same tenant.")]
```

Comment:
This test has to be "skipped" because any test that uses the "Add-AzVirtualNetworkPeering" commandlet throws the following error message:

"Authentication failed for auxiliary token: The '1' auxiliary tokens contains duplicates which are from the same tenant."

The only other test case that uses this commandlet is TestVirtualNetworkPeeringCRUD() and it is also marked as "skip" for the same reason.